### DISCUSSION TITLE
AltGr as Alt key

### DISCUSSION THREAD

**Discussion by MyGitHubBlueberry** [hyprwm/Hyprland#5985] [Q&A] [answered]
Hello, I'm new to hyrpland.
I'm using `us` and `ua` layouts and `right alt` on ua layout is `AltGr` or `ISO_Level3_Shift` , as `wev` suggests.
I'm having hard time making it behave like regular `Alt`.
I tried 
`
    kb_options = ISO_Level3_Shift:alt, AltGr:alt
`
And other same options. Nothing works for me. 
Any help appreciated.

**Comment by MyGitHubBlueberry** [comment, marked as answer]
It's `MOD5` key, never mind...

**Reply by wayfarren** [reply]
Hi, can you list the exact steps you used to get around this issue? I can't for the life of me figure out how to get this working.

**Reply by sudiplun** [reply]
As Example 
`bind = MOD5, K, exec, kitty`

**Reply by wayfarren** [reply]
But specifically for the purpose of remapping a key? Making AltGr interpreted at Alt

**Reply by sudiplun** [reply]
`kb_options = lv3:lalt_switch` # Remap `alt` to `alt gr`
but
`kb_options = lalt_switch:lv3` # doesn't Remap `alt gr` to `alt`

Doesn't know why?

**Reply by wayfarren** [reply]
I just tried experimenting with it too. I can't get it to work either.
